runtime/doc/usr_05.txt

```diff
@@ -387,8 +383,91 @@ Only files of this filetype will use the settings from the plugin.  The <name>
 part of the plugin file doesn't matter, you can use it to have several plugins
 for the same filetype.  Note that it must end in ".vim".
 
+USING PLUGIN BUNDLES                                      *plugin-bundles*
+
+The manual method described above has the problem that things can get messy
+once you install a few plugins, specially if they include more than one file
+(which is common). A way to solve this is to install plugins in their own
+directories and then add the path of those to the 'runtimepath' setting (so
+Vim will use the files from those directories when looking for plugins). In
+practice, this means that instead of having a |vimfiles| tree like >
+
+	~.vim/
+	    plugin/
+		a.vim
+	    ftplugin/
+		b.vim
+		e.vim
+	    syntax/
+		b.vim
+
+you can have something like this >
+
+	~.vim/
+	    bundles/
+		a/
+		    plugin/
+			a.vim
+		b/
+		    ftplugin/
+			b.vim
+		    syntax/
+			b.vim
+		e/
+		    ftplugin/
+			e.vim
+
+This method allows the plugins to be distributed as their own trees, so they
+can be installed without affecting other plugins. This is very helpful when
+synchronizing with a version control repository.
+
+So supposing the user wants to install a plugin that's distributed as this tree: >
+
+	autoload/
+	    awesome.vim
+	plugin/
+	    awesome.vim
+	doc/
+	    awesome.txt
+
+from http://github.com/developer/awesome-vim, they can do so by (assuming there
+is no "bundles" directory)
+
+    1. creating the plugin directory >
+	$ mkdir -p ~/.vim/bundles/awesome-vim
+<    2. copying the files into the "bundles/awesome-vim/" folder >
+	$ cp awesome-vim/* ~/.vim/bundles/awesome-vim/
+<    3. adding >
+
+	set runtimepath+=~/.vim/bundles/awesome-vim
```

Comment:
29.09.2015, 01:36, "Felipe Morales" <notifications@github.com>:
> In runtime/doc/usr_05.txt:
>
>> + awesome.vim > + plugin/ > + awesome.vim > + doc/ > + awesome.txt > + > +from http://github.com/developer/awesome-vim, they can do so by (assuming there > +is no "bundles" directory) > + > + 1. creating the plugin directory > > + $ mkdir -p ~/.vim/bundles/awesome-vim > +< 2. copying the files into the "bundles/awesome-vim/" folder > > + $ cp awesome-vim/* ~/.vim/bundles/awesome-vim/ > +< 3. adding > > + > + set runtimepath+=~/.vim/bundles/awesome-vim
>
> Yeah, I thought it looked weird too close to the surrounding lines since this is not prefixed by a $, like in the other cases. It looks clearer this way to me. I could give more room to the previous codeblocks, but I won't remove the empty lines in this one.

It does not matter as long as you are consistent. If you think that less space here looks bad, add more space there.

>
> —
> Reply to this email directly or view it on GitHub.